### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getMarker' of undefined

### ISSUE THREAD

**Issue by jungangl** [JunoLab/atom-ink#157]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.23.1 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.11.6
**Thrown From**: [ink](https://github.com/JunoLab/atom-ink) package 0.7.11

### Stack Trace

Uncaught TypeError: Cannot read property 'getMarker' of undefined

```
At /Users/jungangl/Documents/Programming Languages/Julia/Atom.app/Contents/Resources/app/src/decoration-manager.js:193

TypeError: Cannot read property 'getMarker' of undefined
    at DecorationManager.decorateMarker (/Users/jungangl/Documents/Programming Languages/Julia/Atom.app/Contents/Resources/app/src/decoration-manager.js:193:69)
    at TextEditor.decorateMarker (/Users/jungangl/Documents/Programming Languages/Julia/Atom.app/Contents/Resources/app/src/text-editor.js:2109:41)
    at Result.decorateMarker (/packages/ink/lib/editor/result.js:74:35)
    at Result.attach (/packages/ink/lib/editor/result.js:45:10)
    at /packages/ink/lib/editor/result.js:24:10)
    at /packages/julia-client/lib/runtime/evaluation.coffee:35:33
```

### Commands

```
  2x -2:10.1.0 core:save (input.hidden-input)
     -2:09.3.0 inline:clear-current (input.hidden-input)
     -2:09.3.0 editor:consolidate-selections (input.hidden-input)
     -2:09.3.0 core:cancel (input.hidden-input)
  2x -2:06.8.0 core:save (input.hidden-input)
  2x -2:02.0 core:backspace (input.hidden-input)
  2x -2:01.2.0 core:save (input.hidden-input)
     -2:00.2.0 julia-client:run-file (input.hidden-input)
     -2:00.2.0 autocomplete-plus:cancel (atom-text-editor.editor)
     -2:00.2.0 inline-results:clear-all (atom-text-editor.editor)
  2x -1:53.9.0 core:save (input.hidden-input)
     -1:52.8.0 julia-client:run-file (input.hidden-input)
     -1:52.8.0 autocomplete-plus:cancel (atom-text-editor.editor)
     -1:52.8.0 inline-results:clear-all (atom-text-editor.editor)
     -0:11.5.0 julia-client:run-block (input.hidden-input)
     -0:11.5.0 autocomplete-plus:cancel (atom-text-editor.editor.is-focused.has-selection)
```

### Non-Core Packages

```
ink 0.7.11 
julia-client 0.6.6 
language-julia 0.14.0 
latex-completions 0.3.3 
print-atom 0.1.1 
tool-bar 1.1.0 
uber-juno 0.1.2 
```

**Comment by pfitzseb** [comment]
Can you give some more info on this? In particular, can you reproduce it by evaling into a file, clearing the inline result and then evaling again?

**Comment by YongHee-Kim** [comment]
I'm running into this issue as well 
I've tried to reproduce this but failed to do so. It just happens sometimes...

But it's something to do with re-evaluating code in a file that is already included.
Here is reproduce steps I have 

1. Evaluate code block with `ctrl-enter` with bunch of `include`
``` julia
using Missings
......
include("worksheet.jl")

```
2. Evaluate line of code in `workbook.jl` with `ctrl-enter`

![bug_report](https://user-images.githubusercontent.com/7087745/34551739-dd5a2504-f15f-11e7-853c-0269064a1671.gif)

[This is a package](https://github.com/YongHee-Kim/EzExcel.jl) I'm running in GIF animation.

```
Atom: 1.23.2 x64

Julia Version 0.6.1
Commit 0d7248e2ff* (2017-10-24 22:15 UTC)
Platform Info:
  OS: Windows
 (x86_64-w64-mingw32)
  CPU: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, haswell)
```

**Comment by pfitzseb** [comment]
I have a hunch as to what might cause this and pushed a possible fix. Still can't reproduce the issue though, so no idea if it works.

**Comment by hpoit** [comment]
This is strange, everything passed just yesterday, but not today (no code changed).
![image](https://user-images.githubusercontent.com/3711100/34942516-1a2c97a0-f9df-11e7-9f1a-693e13eee1a5.png)

The code https://github.com/hpoit/Weiqi.jl/blob/master/test/board.jl

I closed Atom and re-executed everything again, and 
![image](https://user-images.githubusercontent.com/3711100/34944226-f2ee31ce-f9e5-11e7-9b87-9b77db5bc58e.png)

Because I am new to programming and Atom, in the first pic I executed the test/board.jl before anything else, got that message, and then executed Weiqi.jl, src/board.jl, and src/chineserules.jl, but continued getting the error. Maybe the sequence of execution caused the error? Not sure.
